src/handlers/security-headers-handler.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  securityHeadersHandler: (req, res, next) => {
+    res.set({
+      "Strict-Transport-Security": "max-age=31536000",
+      "Content-Security-Policy": "default-src 'self';",
```

Comment:
Thinking this might block assets if they're hosted on a different domain?

Reply:
Confirmed with Toby we should add `*.identity.gov.uk` here so the assets don't get blocked on build and above where the assets are externally hosted